scene/gui/item_list.cpp

```diff
@@ -1617,6 +1485,150 @@ void ItemList::set_text_overrun_behavior(TextServer::OverrunBehavior p_behavior)
 	}
 }
 
+void ItemList::update_auto_height() {
+	if (!shape_changed) {
+		return;
+	}
+
+	Ref<StyleBox> bg = get_theme_stylebox(SNAME("bg"));
```

Comment:
Unused variable?

Reply:
`bg` is superfluous, yeah, but `size` is used at line 1556 and 1605 for this PR.